IronFrame.Test/Utilities/JobObjectTests.cs

```diff
@@ -223,6 +224,62 @@ public void WhenMemoryIsNotLimited_GetMemoryPeakReturnsZero()
             }
         }
 
+        public class JobCpuLimits : IDisposable
+        {
+            JobObject jobObject;
+            JobObject jobObject2;
+
+
+            public JobCpuLimits()
+            {
+                jobObject = new JobObject();
+                jobObject2 = new JobObject();
+            }
+
+            public void Dispose()
+            {
+                jobObject.Dispose();
+                jobObject2.Dispose();
+            }
+
+            [Fact]
+            public void CanLimitCpu()
+            {
+                jobObject.SetJobCpuLimit(1);
+                jobObject2.SetJobCpuLimit(9);
+
+                var thread1 = new Thread(() =>
```

Comment:
Instead of launching these two threads, you could just launch the two processes, and then wait for both to exit.

```
var process1 = ExecuteInJob(...);
var process2 = ExecuteInJob(...);

process1.WaitForExit();
process2.WaitForExit();

Also, running them in a thread like that will not report any exceptions to the test runner (at best) and will crash the test process (at worst).